### DISCUSSION TITLE
35th open maintainers call?

### DISCUSSION THREAD

**Discussion by dhuseby** [libp2p/rust-libp2p#5136] [General]
I noticed that the rust-libp2p open maintainers call was on the calendar for yesterday. It's an artifact from with @mxinden ran the meetings. It happened at 4am my time so I was unable to attend. Did any of you attend? How many attended? @thomaseizinger did you attend?

I'm volunteering to organize the next one and I'm curious about which times work best for everybody. Do you like the current time slot of 11:00 UTC? That's very Europe friendly but not so North America friendly. Maybe we need to start a rotating community call to cover the globe. Say 11:00 UTC (EU), then 11:00 UTC-8 (Americas), then 11:00 UTC+8 (APAC)?

What are your thoughts?

**Comment by thomaseizinger** [comment]
I was travelling yesterday and somewhere over the pacific ocean at that point 😁

Historically, the maintainers were in EU and APAC hence the current timezone. Feel free to move it to a NA and EU friendly one! :)

**Reply by dhuseby** [reply]
@thomaseizinger thanks, will do
